$OpenBSD: patch-cpio_bsdcpio_1,v 1.3 2016/01/30 23:34:44 naddy Exp $

Match installed name.

CVE-2015-2304
Commit 5935715
    Add ARCHIVE_EXTRACT_SECURE_NOABSOLUTEPATHS option
    This fixes a directory traversal in the cpio tool.

--- cpio/bsdcpio.1.orig	Mon Jan 14 02:43:45 2013
+++ cpio/bsdcpio.1	Sat Jan 30 21:33:16 2016
@@ -28,7 +28,7 @@
 .Dt CPIO 1
 .Os
 .Sh NAME
-.Nm cpio
+.Nm bsdcpio
 .Nd copy files to and from archives
 .Sh SYNOPSIS
 .Nm
@@ -156,7 +156,8 @@ See above for description.
 .It Fl Fl insecure
 (i and p mode only)
 Disable security checks during extraction or copying.
-This allows extraction via symbolic links and path names containing
+This allows extraction via symbolic links, absolute paths,
+and path names containing
 .Sq ..
 in the name.
 .It Fl J , Fl Fl xz
